.github/workflows/generate-package-versions.yml

```diff
@@ -99,4 +99,30 @@ jobs:
           base: main
           title: "chore: update ${{ env.VENV_NAME }} latest version to ${{ env.NEW_LATEST }}"
           labels: changelog/no-changelog
-          body-path: .github/PULL_REQUEST_TEMPLATE.md
+          body: |
```

Comment:
should we remove the .github/PULL_REQUEST_TEMPLATE.md file if we're not using it anymore?